database/README.md

```diff
@@ -433,6 +433,19 @@ Stores the best and second best solution quality (score) of every auction promis
 Indexes:
 - PRIMARY KEY: btree(`auction_id`)
 
+### reference\_scores
+
+Stores the reference score per solver, defined as the difference between the total score for the auction and the combined score of all winning solutions submitted by that solver for the same auction.
+
+ Column           | Type     | Nullable | Details
+------------------|----------|----------|--------
+ auction\_id      | bigint   | not null | id of the auction the scores belong to
+ solver           | bytea    | not null | public address of the winning solver
+ reference\_score | numeric  | not null | score of the runner up solver. If only 1 solver submitted a valid solution this value is 0.
```

Comment:
This is the definition of the old reference score. Given that this table stores only one non-obvious thing which is explained in the table summary we probably don't need to repeat the explanation here.